rules/TypeDeclaration/PhpDocParser/ReturnPhpDocDecorator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\TypeDeclaration\PhpDocParser;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Function_;
+use PHPStan\PhpDocParser\Ast\PhpDoc\ReturnTagValueNode;
+use PHPStan\Type\ArrayType;
+use Rector\BetterPhpDocParser\PhpDocInfo\PhpDocInfoFactory;
+use Rector\Comments\NodeDocBlock\DocBlockUpdater;
+use Rector\StaticTypeMapper\StaticTypeMapper;
+
+final readonly class ReturnPhpDocDecorator
```

Comment:
there is already `PhpDocTypeChanger` service for it

https://github.com/rectorphp/rector-src/blob/e09d82c508da47d736599a57f3a02cae48979027/src/BetterPhpDocParser/PhpDocManipulator/PhpDocTypeChanger.php#L99

Reply:
Good point :+1: